ascii_graph.py

```diff
@@ -0,0 +1,27 @@
+from sympy import symbols
+from sympy.plotting import textplot
+
+import argparse
+import conda.plugins
+
+
+def ascii_graph(argv: list):
+    parser = argparse.ArgumentParser("conda ascii-graph")
+
+    parser.add_argument("x", type=float, help="First coordinate to graph")
+    parser.add_argument("y", type=float, help="Second coordinate to graph")
+    parser.add_argument("z", type=float, help="Third coordinate to graph")
+
+    args = parser.parse_args(argv)
+
+    s = symbols('s')
+    textplot(s**args.x,args.y,args.z)
```

Comment:
Is there value in enabling pre-commit for this repo?

Reply:
Yes, I think so. I'll create an issue for it.

Reply:
https://github.com/conda/conda-plugin-template/issues/12